src/partials/connectors-popular.hbs

```diff
@@ -0,0 +1,134 @@
+<section class="connectors-popular">
+  <header class="header-connectors-title">Popular Connectors</header>
+  <div class="flex popular-cards">
+    <a class="card" href="//docs.mulesoft.com/connectors/salesforce/salesforce-connector">
+      <img
+        alt="Salesforce"
+        class="card-img"
+        height="50"
+        src="/_/img/logos/salesforce.png"
```

Comment:
@runnabro When making a reference to an image from a template, please use `{{uiRootPath}}` instead of `/_`.